lib/sdk/panel/utils.js

```diff
@@ -295,11 +307,27 @@ function make(document) {
     events.emit(type, { subject: panel })
   }
 
+  function onPanelClick(event) {
```

Comment:
This does not captures anything from the scope so please this function top level instead.